examples/event-loop.js

```diff
@@ -57,7 +57,7 @@ libui.startLoop();
 const linebreak = process.platform === "win32" ? '\r\n' : '\n';
```

Comment:
We can use `require('path').sep` to get the correct line break for the platform.

Reply:
Oh yes, sorry...

> It's also interesting that textarea.getText() returns the text with "\n" (also on windows), but setText() requires the platform specific type.

That's strange, we should investigate if it's intentional or not.